omega/omega-spring-tx/src/test/java/org/apache/servicecomb/pack/omega/transaction/spring/MessageConfig.java

```diff
@@ -19,48 +19,59 @@
-import org.apache.servicecomb.pack.omega.context.IdGenerator;
-import org.apache.servicecomb.pack.omega.context.OmegaContext;
+import org.apache.servicecomb.pack.omega.context.CallbackContextManager;
+import org.apache.servicecomb.pack.omega.context.OmegaContextManager;
+import org.apache.servicecomb.pack.omega.context.ParameterContextManager;
+import org.apache.servicecomb.pack.omega.context.TransactionType;
 import org.apache.servicecomb.pack.omega.transaction.AlphaResponse;
+import org.apache.servicecomb.pack.omega.transaction.CompensationMessageHandler;
+import org.apache.servicecomb.pack.omega.transaction.SagaMessageHandler;
 import org.apache.servicecomb.pack.omega.transaction.SagaMessageSender;
-import org.apache.servicecomb.pack.omega.transaction.TxEvent;
-import org.apache.servicecomb.pack.omega.transaction.tcc.DefaultParametersContext;
-import org.apache.servicecomb.pack.omega.transaction.tcc.ParametersContext;
+import org.apache.servicecomb.pack.omega.transaction.SagaStartAspect;
+import org.apache.servicecomb.pack.omega.transaction.TransactionAspect;
+import org.apache.servicecomb.pack.omega.transaction.tcc.CoordinateMessageHandler;
+import org.apache.servicecomb.pack.omega.transaction.tcc.TccMessageHandler;
 import org.apache.servicecomb.pack.omega.transaction.tcc.TccMessageSender;
-import org.apache.servicecomb.pack.omega.transaction.tcc.events.*;
-import org.mockito.Mockito;
+import org.apache.servicecomb.pack.omega.transaction.tcc.TccParticipatorAspect;
+import org.apache.servicecomb.pack.omega.transaction.tcc.TccStartAspect;
+import org.apache.servicecomb.pack.omega.transaction.tcc.events.CoordinatedEvent;
+import org.apache.servicecomb.pack.omega.transaction.tcc.events.ParticipationEndedEvent;
+import org.apache.servicecomb.pack.omega.transaction.tcc.events.ParticipationStartedEvent;
+import org.apache.servicecomb.pack.omega.transaction.tcc.events.TccEndedEvent;
+import org.apache.servicecomb.pack.omega.transaction.tcc.events.TccStartedEvent;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 public class MessageConfig {
   private final List<String> messages = new ArrayList<>();
 
-  @Bean
-  IdGenerator<String> idGenerator() {
-    return Mockito.mock(IdGenerator.class);
-  }
-
-  @Bean(name = "compensationContext")
-  CallbackContext recoveryCompensationContext(OmegaContext omegaContext) {
-    return new CallbackContext(omegaContext);
-  }
-
-  @Bean(name = "coordinateContext")
-  CallbackContext coordinateContext(OmegaContext omegaContext) {
-    return new CallbackContext(omegaContext);
-  }
-
-  @Bean
-  ParametersContext parametersContext() {
-    return new DefaultParametersContext();
-  }
-
-  @Bean
-  OmegaContext omegaContext(IdGenerator<String> idGenerator) {
-    return new OmegaContext(idGenerator);
-  }
+//  @Bean
```

Comment:
If we don't use the code we can just remove the code, don't comment them out.

Reply:
ok let me clear up